.github/workflows/build.yml

```diff
@@ -55,3 +55,17 @@ jobs:
           annotations: ${{ steps.extract_metadata.outputs.annotations }}
           cache-from: type=gha,scope=ubuntu
           cache-to: type=gha,mode=max,scope=ubuntu
+
+      - name: Extract binaries from image
+        if: startsWith(github.ref, 'refs/tags/v')
+        uses: shrink/actions-docker-extract@v3
+        with:
+          image: ${{ steps.build_image.outputs.imageid }}
+          path: /opt/bundle/.
+          destination: out
+
```

Comment:
Maybe a checksum step to generate a `.sha256sum` file for each binary?

It would be nice, but not a requirement.

Reply:
How about a single `SHA256SUMS.txt` file?

Reply:
It also works, don't need `.txt` extensio.

But I like individual sha256sum file per binary, so you can download like

```shell
curl https://example.com/file
curl https://example.com/file.sha256
```

But it's up to you.

Reply:
Done!